hygen/_templates/layer/new/prompt.js

```diff
@@ -56,7 +56,7 @@ const prompt = async ({ prompter, args }) => {
 
   // Detect what kind of layer we need (CartoSQLLayer, CartoBQTilerLayer)
   const selectedSourceFileContent = readFile(`src/data/sources/${answers.source_file}.js`);
-  const res = /(?:type: ')(?<type>[\w]*)(?:')/gi.exec(selectedSourceFileContent);
+  const res = /(?:type: ')(?<type>sql|bq*)(?:')/g.exec(selectedSourceFileContent);
```

Comment:
@AdriSolid we will have to consider this if we make the 'bq' to 'bigquery' refactor later

Reply:
Yep, that's right. Maybe it'd be nice in a near future to refactor hygen a bit to use the same constants to define source types, it's also in other generators.